types/twine-sugarcube/wiki.d.ts

```diff
@@ -1,22 +1,15 @@
+export type OutputDestination = DocumentFragment | HTMLElement | JQuery;
+export interface WikifierOptions {
+    ignoreTerminatorCase?: boolean;
+    profile: string;
+}
+
 export interface WikifierAPI {
-    createExternalLink(destination: string, url: string, text: string): HTMLElement;
-    createInternalLink(destination: string, passage: string, text: string, callback: () => void): HTMLElement;
-    evalExpression(code: string): any;
-    evalStatements(code: string): any;
-    /**
-     * @see State.getVar
-     */
-    getValue(varName: string): any;
-    /**
-     * @see Story.has
-     * @param link link
-     */
-    isExternalLink(link: string): boolean;
-    parse(text: string): any;
-    /**
-     * @see State.setVar
-     */
-    setValue(varName: string, value: any): boolean;
+    new(destination: OutputDestination | null, source: string, options?: WikifierOptions): unknown;
+
+	createExternalLink(destination: OutputDestination, url: string, text: string): HTMLElement;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/wiki.d.ts;
  Line: 10;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.